core/server/master/src/main/java/alluxio/master/block/DefaultBlockMaster.java

```diff
@@ -689,6 +696,54 @@ public BlockInfo getBlockInfo(long blockId) throws BlockInfoException, Unavailab
     return ret;
   }
 
+  private MasterWorkerInfo lookupWorker(WorkerNetAddress workerNetAddress) {
```

Comment:
Maybe this should be named `findUnregisteredWorker()`, so it is clear it will not search `mWorkers`.

Could you javadoc this method to describe what it does?